src/main/java/teammates/common/util/Const.java

```diff
@@ -875,6 +875,7 @@
         public static final String STUDENT_GOOGLEID_RESET_FAIL = "An error occurred when trying to reset student's google id";
         
         public static final String STUDENT_EDITED = "The student has been edited successfully";
+        public static final String STUDENT_NOT_FOUND = "The student you tried to edit does not exist. If the student was created during the last few minutes, try again in a few more minutes as the student may be being saved.";
```

Comment:
It was done in the related PR where the second sentence was extracted to be appended onto the other error messages.
I believe there shouldn't be a need to duplicate the change here when that PR is planned to be merged into this one.